Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -246,7 +249,9 @@ def test_module(client: Client, *args) -> Tuple[str, dict, dict]:
     client.run_parameters_validations()
 
     for service in client.services:
-        client.build_iterator(service, client.indicator_type)
+        # if there are risk rules, select the first one for test
+        risk_rule = client.risk_rule[0] if client.risk_rule else None
```

Comment:
why dont we test all ?

Reply:
the validation of all configured risk rules is verified in run_parameters_validations(). 
Running the fetch flow for each one of them takes time ... I think it is unnecessary and we can be satisfied with running the flow for one of them in order to check the correctness of the instance and the API calls